### DISCUSSION TITLE
Default value not picked by trigger

### DISCUSSION THREAD

**Discussion by paulvonber** [directus/directus#17610] [Archive]
Is this is expected behaviour that triggers in the flows doesn't pick up default field value? 
And if yes is it there anyway to get over it? I mean to somehow include default value if it's not changed?

**Comment by br41nslug** [comment]
Default values are handled by the database so the most reliable way to get the default values is to read the item from the database using it's ID. This only works for `action` flows not for `filters` as the data has not been saved yet when the `filter` fires.

**Reply by paulvonber** [reply]
not sure if I fully got your point, could you maybe drop some hint how the payload would look like?
